### DISCUSSION TITLE
Add tooltip to show full exact timestamp of comment revision history

### DISCUSSION THREAD

**Discussion by nipunn1313** [community/community#35217] [Pull Requests]
![GIF 10-5-22 at 9 46 12 AM](https://user-images.githubusercontent.com/1300387/194116657-8701cf14-4b06-49b9-8dba-83baa5b2782e.gif)

GIF here showing where I'd like to be able to see the exact timestamp.
The pattern of using a tooltip exists in many other places on github's PR viewer.

**Comment by ogrisel** [comment]
I also find the coarse-grained time display of comments a bit annoying: for instance, when I read a comment was posted "last week", I cannot know whether if the author of that comment wrote it before or after an IRL discussion or a discussion in another linked issue that happened at the end of the week. If this comment is inconsistent with the conclusions of that other discussion, I cannot assess if this is because of a real misunderstanding or because the comment was written before building a common understanding in the subsequent discussion.
